tools/cli/main.cpp

```diff
@@ -163,19 +164,59 @@ static int inject_by_frida(int pid, const char *inject_path, const char *arg)
 	return 0;
 }
 
-static std::pair<std::string, std::vector<std::string> >
+static std::tuple<std::string, std::vector<std::string>, std::vector<const char *>>
 extract_path_and_args(const argparse::ArgumentParser &parser)
 {
 	std::vector<std::string> items;
+	std::vector<const char *> env_args;
 	try {
 		items = parser.get<std::vector<std::string> >("COMMAND");
+
+		if (parser.is_subcommand_used("load") || parser.is_subcommand_used("start")) {
```

Comment:
`extract_path_and_args` is receiving a sub parser, not the full parser. So there is no sub commands under it, and these `parser.is_subcommand_used` calls will always throw exception